features/object_mapper/creating/README.md

```diff
@@ -0,0 +1,168 @@
+- `name` : the name of the table in the database.
+- `caseSensitiveKeyspace` : whether the keyspace name is [case sensitive].
+- `caseSensitiveTable` : whether the table name is [case sensitive].
+- `readConsistency` : the [consistency level] that will be used on each get operation in the mapper. (if unspecified, it defaults to the cluster-wide setting)
+- `writeConsistency` : the [consistency level] that will be used on each save, or delete operation in the mapper. (if unspecified, it defaults to the cluster-wide setting)
+
+A class annotated with `@Table` must provide a default constructor.
+
+[table]:http://docs.datastax.com/en/drivers/java/2.1/com/datastax/driver/mapping/annotations/Table.html
+[case sensitive]:http://docs.datastax.com/en/cql/3.2/cql/cql_reference/ucase-lcase_r.html
+[consistency level]:http://docs.datastax.com/en/drivers/java/2.1/com/datastax/driver/core/ConsistencyLevel.html
+
+--------------
+
+[@Column][column] can be used on each field of a mapped object to specify the column name in the database. `caseSensitive` can be used for [case sensitive] columns. If `@Column` is not specified, the name of the field will be used.
+
+[column]:http://docs.datastax.com/en/drivers/java/2.1/com/datastax/driver/mapping/annotations/Column.html
+
+--------------
+ 
+[@PartitionKey][pk] and [@ClusteringColumn][cc] are used to indicate the [partition key][pks] and [clustering columns][pks]. In the case of a composite partition key or multiple clustering columns, the integer value indicates the position of the column in the key:
+ 
+```java
+@PartitionKey(0)
+private String countryCode;
+@PartitionKey(1)
+private String areaCode;
+```
+
+[pk]:http://docs.datastax.com/en/drivers/java/2.1/com/datastax/driver/mapping/annotations/PartitionKey.html
+[cc]:http://docs.datastax.com/en/drivers/java/2.1/com/datastax/driver/mapping/annotations/ClusteringColumn.html
+[pks]:http://thelastpickle.com/blog/2013/01/11/primary-keys-in-cql.html
+
+--------------
+
+[@Enumerated][enum] is used to reference a field typed as a Java enumeration into the database. Since *Cassandra* does not support enumerations, the object mapper defines to ways to persist a `enum` value :
+
+- `EnumType.ORDINAL` : the value is persisted as a C\*'s `int` and its value is its position in the `enum`.
+- `EnumType.STRING` : the value is persisted as a C\*'s `text` and its value is its name in the `enum`.
+
+According to the chosen type, the matching database column must be the same type.
+
+```java
+@Enumerated(EnumType.ORDINAL)
+MyEnum enum;
+```
+
+[enum]:http://docs.datastax.com/en/drivers/java/2.1/com/datastax/driver/mapping/annotations/Enumerated.html
+
+--------------
+
+[@Computed][computed] can be used on fields that are the result of a computation on the *Cassandra* side. Typically a function call. Native function in *Cassandra* like `writetime()` or [User Defined Functions] are supported.
+ 
+```java
+@Computed(formula = "ttl(name)")
+Integer ttl;
```

Comment:
I've just realized that you can't omit the attribute name from the declaration. I wanted the possibility to write this:
```java
@Computed("ttl(name)")
```
I've pushed a quick fix to the 2.1 branch to rename `formula` to the default `value`. I'll update this doc.